static/openapi/p3-spec.json

```diff
@@ -0,0 +1,3479 @@
+                      "description": "The type of device to simulate.",
+                      "example": "Desktop"
+                    },
+                    "networkCondition": {
+                      "type": "string",
+                      "enum": [
+                        "Slow4G",
+                        "Regular3G"
+                      ],
+                      "description": "The network condition to simulate.",
+                      "example": "Regular3G"
+                    },
+                    "url": {
+                      "type": "string",
+                      "description": "The URL to test.",
+                      "example": "https://something.com/something"
+                    },
+                    "viewport": {
+                      "type": "object",
+                      "properties": {
+                        "height": {
+                          "type": "integer",
+                          "description": "The viewport height in pixels.",
+                          "example": 1080
+                        },
+                        "width": {
+                          "type": "integer",
+                          "description": "The viewport width in pixels.",
+                          "example": 1920
+                        }
+                      },
+                      "required": [
+                        "height",
+                        "width"
+                      ]
+                    },
+                    "headers": {
+                      "type": "object",
+                      "additionalProperties": {
+                        "type": "string"
+                      },
+                      "description": "Any additional headers to send in the request."
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "responses": {
+            "200": {
+              "description": "PPM Job successfully created.",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "The unique identifier of the created PPM job."
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "400": {
+              "description": "Bad request. Possible issue with the job data provided."
+            },
+            "500": {
+              "description": "Internal server error."
+            }
+          }
+        }
+      },
+      "/api/p3/v1/ppm/vitals/{percentile}": {
+        "get": {
+          "tags": [
+            "PPM"
+          ],
+          "summary": "Get Web Vitals Metrics Percentile",
+          "description": "Retrieves web vitals metrics percentile data based on the provided datetime range and pagination.",
```

Comment:
I am not sure how this api works based on the parameters.

You pass in 75.  What is it supposed to return?

Reply:
I would not bother mentioning the "pagination" in the description.  Focus more on what is this API going to do for me?  It looks like some kind of metrics, I don't know what vitals is, don't understand percentile data thing.